src/index.js

```diff
@@ -128,7 +128,7 @@ module.exports = class WyzeAPI {
       throw e
     }
     if (result.data.msg) {
-      if (result.data.msg == "DeviceIsOffline") { return result } else
+      if (result.data.msg == "DeviceIsOffline" || result.data.msg != "SUCCESS") { return result } else
```

Comment:
```suggestion
      if (result.data.msg == "DeviceIsOffline" || result.data.msg == "SUCCESS") { return result } else
```

thanks for the PR - small typo to fix before merging